components/Form.js

```diff
@@ -0,0 +1,59 @@
+import { useRouter } from "next/router";
+import styled from "styled-components";
+import { useState } from "react";
+
+const Label = styled.label`
+  font-weight: bold;
+`;
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export default function Form({ onSubmit, entrySucessful }) {
+  const router = useRouter();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    onSubmit(data);
+    entrySucessful();
+  }
+
+  return (
+    <StyledForm onSubmit={handleSubmit}>
+      <Label htmlFor="name">Name</Label>
+      <input type="text" id="name" name="name" required maxLength={150} />
+      <Label htmlFor="botanical_name">botanical name</Label>
+      <input
+        type="text"
+        id="botanical_name"
+        name="botanical_name"
+        required
+        maxLength={150}
+      />
+      <Label htmlFor="water_need">water need</Label>
+      <input
+        type="text"
+        id="water_need"
+        name="water_need"
+        required
+        maxLength={10}
+      />
+      <Label htmlFor="image">Image</Label>
+      <input type="text" id="image" name="image" maxLength={150} />
```

Comment:
this is pretty hacky. Usually you would include a proper image upload here. The way that you are planning this image input won't work out this easily. 
First of all, the user needs specific information that they need to put in the whole url address (https://....) 
That best to include in the label and also in a placeholder. (The placeholder disappears after the user starts typing, thats why the label is better)

Then there is a another restriction: Next.js only allows images in the app that are from a know source. You need to place these know sources in the next.config. [Check this link for more info on this.](https://nextjs.org/docs/pages/api-reference/components/image#configuration-options)

So you need to let the user know that they can only upload images from very specific sources that you configured in the app before.
This is all very user unfriendly :D So I would, for now, just add an example image to every form entry and only develop this feature fully once you tackle the image upload with cloudinary. 
Please let me know if you have any questions about this :)